src/lambda/registrations/index.js

```diff
@@ -62,6 +62,8 @@ const handler = async (event, context) => {
     }
   }
 
+  console.log(params)
+
```

Comment:
Still wondering if this should be enabled by default?

Reply:
This has been removed.